anndata/tests/test_base.py

```diff
@@ -118,6 +118,15 @@ def test_attr_deletion():
     assert_equal(full, empty, exact=True)
 
 
+def test_not_view_after_deletion():
+    full = gen_adata((30, 30))
+    for attr in ["X", "obs", "var", "obsm", "varm", "obsp", "varp", "layers", "uns"]:
+        view = full[:, :20]
+        assert view.is_view
+        delattr(view, attr)
+        assert not view.is_view
```

Comment:
Could this instead be a modification of `test_view_delattr` in `test_views.py`?

Reply:
In general, I think `test_base.py` covers too much API surface, and has too much in it. When possible, try to add tests in more specific files. Should make it easier to find tests in the future.